ecs-agent/daemonimages/csidriver/util/fs/fs_unsupported.go

```diff
@@ -20,11 +20,6 @@ import (
 	"fmt"
 )
 
-type UsageInfo struct {
```

Comment:
Was this unused and so we are deleting it?

Reply:
Move this to a common file.